network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ConnectablePositionImpl.java

```diff
@@ -40,7 +52,21 @@ public Optional<String> getName() {
 
         @Override
         public Feeder setName(String name) {
-            getAttributes().setLabel(Objects.requireNonNull(name));
```

Comment:
you can keep Objects.requireNonNull at the beginning

Reply:
This code is outdated.